Providers/nxOMSAutomationWorker/automationworker/scripts/onboarding.py

```diff
@@ -436,15 +441,17 @@ def environment_prerequisite_validation():
 
 
 def get_options_and_arguments():
-    parser = OptionParser(usage="usage: %prog [--register, --deregister] -e endpoint -k key -g groupname -w workspaceid",
-                          version="%prog " + str(configuration.get_worker_version()))
+    parser = OptionParser(
+        usage="usage: %prog [--register, --deregister] -e endpoint -k key -g groupname -w workspaceid [-r gpgkeyring]",
+        version="%prog " + str(configuration.get_worker_version()))
     parser.add_option("-e", "--endpoint", dest="registration_endpoint", help="Agent service registration endpoint.")
     parser.add_option("-k", "--key", dest="automation_account_key", help="Automation account primary/secondary key.")
     parser.add_option("-g", "--groupname", dest="hybrid_worker_group_name", help="Hybrid worker group name.")
     parser.add_option("-w", "--workspaceid", dest="workspace_id", help="Workspace id.")
     parser.add_option("-r", "--register", action="store_true", dest="register", default=False)
     parser.add_option("-d", "--deregister", action="store_true", dest="deregister", default=False)
     parser.add_option("-t", "--test", action="store_true", dest="test", default=False)
+    parser.add_option("-r", "--gpg-keyring", dest="gpg_keyring", help="GPG keyring path")
     (options, args) = parser.parse_args()
```

Comment:
There is already a -r earlier, will this be a problem?

Reply:
nice catch, that is a problem